### PR TITLE
Add Webview impl_url for Share API

### PR DESCRIPTION
**Pull request by skyclouds2001**
#### Summary

same as title

#### Related issues

relates to https://github.com/mdn/browser-compat-data/issues/24690

### CODE REVIEW

File: api/Navigator.json [hunk 1]
```diff
@@ -486,7 +486,8 @@
             "safari_ios": "mirror",
             "samsunginternet_android": "mirror",
             "webview_android": {
-              "version_added": false
+              "version_added": false,
+              "impl_url": "https://crbug.com/765923"
```
Comment (caugner) [inline api/Navigator.json:490]:
```suggestion
              "impl_url": "https://crbug.com/40540400"
```

File: api/Navigator.json [hunk 2]
```diff
@@ -4462,7 +4463,8 @@
...
```
Comment (caugner) [inline api/Navigator.json:4467]:
```suggestion
              "impl_url": "https://crbug.com/40540400"
```

**Review by caugner — COMMENTED** [review]
Are we intentionally using the old bug ids here?

**Comment by skyclouds2001** [comment]
> Are we intentionally using the old bug ids here?

I think there is no rule of using which type of bug id.

**Review by caugner — APPROVED** [review]

**Pull request merged**